csrankings-6.csv

```diff
@@ -667,7 +667,7 @@ Mustafa Cenk Gursoy,Syracuse University,http://eng-cs.syr.edu/directory,qBxl76YA
 Mustafa E. Kamasak,Istanbul Technical University,http://web.itu.edu.tr/kamasak,U5UhNHEAAAAJ
 Mustafa Ozdal,Bilkent University,http://www.cs.bilkent.edu.tr/~mustafa.ozdal,mD2TUGQAAAAJ
 Mustafa Özuysal,Izmir Institute of Technology,http://ceng.iyte.edu.tr/people/mustafa-ozuysal,CfT5AZYAAAAJ
-Mustakimur R. Khandaker,University of Georgia,http://www.mustakim.info,pEvjTs8AAAAJ
+Mustakimur Khandaker,University of Georgia,https://www.cs.uga.edu/directory/people/mustakimur-r-khandaker,pEvjTs8AAAAJ

```

Comment:
Please add a duplicate entry for the alias `Mustakimur Rahman Khandaker` and an entry in `dblp-aliases.csv`.

Reply:
I did not understand the comment. Should I keep the change of this pull request but also change in dblp-aliases.csv with Mustakimur Rahman Khandaker Mustakimur Khandaker

Reply:
Yes.

Reply:
Will that be possible to patch two (including) different pull requests: https://github.com/emeryberger/CSrankings/pull/3456

Reply:
Closing this one since it's obsolete.